content/sdk/100.release-note.md

```diff
@@ -2,7 +2,6 @@
 title: "Release Notes"
 description: "Release Notes for SDKs"
 tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "release notes"]
-navigation: false
 ---
```

Comment:
Is there a reason this was removed? There is no content yet here, and I don't want any empty pages for release.